src/db/appsheet_api.py

```diff
@@ -111,10 +119,12 @@ def add_aircraft(self, regno, test=False, description=""):
         try:
             if not self.use_fake_calls:
                 ret = self.sendop(self.config.private_vars["appsheet"]["aircraft_url"], body)
+                appsheet_error_gauge.labels('appsheet add_aircraft error', "").set(0)
```

Comment:
does the set(0) hide earlier errors?  I'm not sure how the 2nd arg is handled.

Reply:
set(0) just sets the value to 0 (initialize).  The thought was to create a trigger so we could alert but again, logs are just as good but better as with logs we don't do the counts here and in the logs.